package.json

```diff
@@ -30,7 +30,7 @@
     "htmlhint": "^1.1.4",
     "jsdoc": "^4.0.2",
     "prettier": "3.0.3",
-    "pyright": "^1.1.324",
+    "pyright": "^1.1.326",
```

Comment:
@eliotwrobson this update resulted in a new error. I think it has to do with the following change from `1.1.325` (https://github.com/microsoft/pyright/releases/tag/1.1.325):

> Behavior Change: Changed interpretation of ... forms of tuples to improve consistency and match mypy's behavior. The type tuple[int, ...] is now interpreted as the union of all tuples of length zero or more that contain only int. Because this is a union, assigning a value of this type to a tuple type with a known (specified) length is now considered a type violation.

Any thoughts on how to fix this?

Reply:
I temporarily downgraded pyright to unblock this PR. I'll open a new one that bumps just this package and @eliotwrobson will push a fix for the type issue.

Reply:
https://github.com/PrairieLearn/PrairieLearn/pull/8431